ci-operator/step-registry/redhat-appstudio/e2e/redhat-appstudio-e2e-commands.sh

```diff
@@ -97,4 +99,11 @@ cd "$(mktemp -d)"
 
 git clone --branch main "https://${GITHUB_TOKEN}@github.com/redhat-appstudio/e2e-tests.git" .
 make ci/prepare/e2e-branch
+# Register Pac host
+pac_host=$(kubectl -n openshift-pipelines get route pipelines-as-code-controller -o jsonpath='{.spec.host}')
+curl -k -X POST -H "Authorization: Bearer ${QE_SPRAYPROXY_TOKEN}" "https://${QE_SPRAYPROXY_HOST}/backends" --data "{\"url\": \"https://$pac_host\"}"
+
 make ci/test/e2e
+
+# Unregister Pac host
+curl -k -X DELETE -H "Authorization: Bearer ${QE_SPRAYPROXY_TOKEN}" "https://${QE_SPRAYPROXY_HOST}/backends" --data "{\"url\": \"https://$pac_host\"}"
```

Comment:
is there some job that unregisters old PaC hosts periodically? (in case this command fails sometimes, leaving records of zombie hosts behind)

Reply:
no, there is not. each time we hope the old Pac hosts are unregistered. But I am considering if there is a better way to unregister the old Pac hosts when the test finish no matter pass or fail.

Reply:
also if we expect devs/QE to be able to run the PaC test locally, we cannot expect everyone to clean up the hosts after themselves. so I think having some periodic job that will do the cleanup automatically is a must have

Reply:
>if we expect devs/QE to be able to run the PaC test locally
they can create/configure their own Github App to run the Pac test, I tested it locally

Reply:
how much user-friendly process is to configure their own GitHub App? is the GitHub App configuration required just once or user would have to do it every time they have a new dev cluster?

note that we don't want devs/QE spending too much time configuring their cluster/dependencies in order to just run/develop tests. it should be ideally all automated

Reply:
we shouldn't expose the QE sprayproxy to everyone, so they have to set up their cluster, including Sprayproxy server,  to run/develop test